src/Command/DbDump.php

```diff
@@ -74,22 +111,39 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $helper = $this->getHelper('question');
-        $question = new ConfirmationQuestion(
-            'We suggest to enable maintenance mode before running this command. Do you want to continue [y/N]?',
-            false
-        );
-        if (!$helper->ask($input, $output, $question) && $input->isInteractive()) {
-            return null;
+        if ($input->isInteractive()) {
+            $helper = $this->getHelper('question');
+
+            $questionParts = [
+                'The db-dump command switches the site to maintenance mode during the dump process.',
+                'Your site will not receive any traffic until the operation completes.',
+                'Do you wish to proceed with this process? (y/N)?',
```

Comment:
@meker12 Could you please rewrite this message with information that we also disabling crons and killing all cron/queue processes after enabling of maintenance mode.